.eslintrc.json

```diff
@@ -16,5 +16,23 @@
   "rules": {
     "padding-line-between-statements": "off",
     "react/jsx-uses-vars": "error"
-  }
+  },
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx"],
```

Comment:
Eventually I think we should integrate these eslint settings for typescript into `@cloudfour/eslint-plugin`, but for now having it here is good so that we can work out any issues we run into